doc/source/user_guide/upf.rst

```diff
@@ -172,7 +172,7 @@ server:
 
     Python UPF Detected
 
-    PYTHON VERSION : 3.6
+    PYTHON VERSION : 3.8
```

Comment:
I dont think you should change that... because that is MAPDL server side. I am not sure which Python version we are shipping in the current/next product. But I do not think it is 3.8.

Reply:
Ah.. this is server side?

Then that output should be Python 3.10. Ansys products dropped Python 3.7 in favor of Python 3.10

Reply:
I just updated it to 3.10